src/otx/recipe/action_detection/x3d_fastrcnn.yaml

```diff
@@ -124,3 +125,5 @@ model:
     lr: 0.005
     momentum: 0.9
     weight_decay: 0.00001
+trainer:
+  precision: 32
```

Comment:
Same question here.

Reply:
This model should be trained by precision32 because this model raises errors when the precision is 16. 
Furthermore, in OTX1.x, this model is also trained by precision 32